.circleci/config.yml

```diff
@@ -1093,11 +1090,7 @@ jobs:
           command: mv ./builds-test ./builds
       - run:
           name: test:e2e:chrome
-          command: |
-            if .circleci/scripts/test-run-e2e.sh
```

Comment:
Do we still use this script somewhere besides these conditions we are removing?

If not, would it make sense to remove the `test-run-e2e.sh` file as well?

Reply:
Great call, removed!